codecov.yml

```diff
@@ -18,13 +18,13 @@ coverage:
             codewhisperer:
                 target: 70%
                 paths:
-                    - src/codewhisperer/*
+                    - packages/core/src/codewhisperer/*
                 flags:
                     - 'codewhisperer'
             amazonqFeatureDev:
```

Comment:
does this correspond to the `flags` thing?

Reply:
Flags scope down coverage reports for different tests/features. You can use codecov projects to report test targets and include flags